core/src/test/scala/pink/cozydev/lucille/ParserSuite.scala

```diff
@@ -127,13 +132,13 @@ class SingleSimpleQuerySuite extends munit.FunSuite {
     assertSingleTerm(r, Field("fieldName42", Boost(Group(And(Term("cats"), Term("dogs"))), 20f)))
   }
 
-  test("parse boost does not parse with trailing 'f' on boost".fail) {
+  test("parse boost does not parse with trailing 'f' on boost") {
     // TODO the trailing `f` is parsing as another term and it should not
```

Comment:
Do we want to ditch this comment?

Reply:
Ah yes! 

Making a suggestion b/c I'm lazy 😅 
```suggestion
```